src/__swaps__/utils/swaps.ts

```diff
@@ -294,48 +290,20 @@ export function valueBasedDecimalFormatter({
     roundedAmount = divWorklet(roundWorklet(mulWorklet(amount, factor)), factor);
   }
 
-  const maximumFractionDigits = () => {
-    // if we're selling max amount, we want to match what's displayed on the balance badge
-    // let's base the decimal places based on that (capped at 6)
-    if (isMaxAmount && assetBalanceDisplay) {
```

Comment:
removing `isMaxAmount` and `assetBalanceDisplay` since we are now just returning the `maxSwappableAmount` for max and using normal formatting.